stackit/internal/services/ske/project/resource.go

```diff
@@ -182,7 +183,24 @@ func (r *projectResource) Delete(ctx context.Context, req resource.DeleteRequest
 	ctx = tflog.SetField(ctx, "project_id", projectId)
 
 	c := r.client
-	_, err := c.DisableService(ctx, projectId).Execute()
+
+	clusters, err := c.ListClusters(ctx, projectId).Execute()
+	if err != nil {
+		core.LogAndAddError(ctx, &resp.Diagnostics, "Error getting the list of clusters", fmt.Sprintf("Calling API: %v", err))
```

Comment:
```suggestion
		core.LogAndAddError(ctx, &resp.Diagnostics, "Error deleting project", fmt.Sprintf("Calling API to get the list of clusters: %v", err))
```

Reply:
The summary argument should always be related to the high-level operation being executed (`Create`, `Delete`, etc) and the details should be related to the specific call that caused an error